### DISCUSSION TITLE
react-dropdown-tree-select in react hook form

### DISCUSSION THREAD

**Discussion by vtharmi** [dowjones/react-dropdown-tree-select#529] [Q&A]
How to use this dropdown tree selector within a react form

**Comment by mrchief** [comment]
@vtharmi is your question about any specific "form" component? There is nothing special you have to do if you're using it in a regular html `form` element. Depending on how your app is setup, you may want to have an [HoC](https://github.com/dowjones/react-dropdown-tree-select/blob/develop/docs/HOC.md#prevent-re-render-on-parent-propstate-changes) to prevent re-renders. If you can provide more details or a codesandbox, it'd make it easier to answer your question.
